game/src/main/kotlin/gg/rsmod/game/plugin/PluginRepository.kt

```diff
@@ -537,6 +542,20 @@ class PluginRepository(val world: World) {
         worldInitPlugins.forEach { logic -> world.executePlugin(world, logic) }
     }
 
+    fun bindSlayer(plugin: Plugin.() -> Unit) {
+        if (slayerPlugin != null) {
+            logger.error("Death plugin is already bound")
+            throw IllegalStateException("Death plugin is already bound")
```

Comment:
Modify this to mention it's a slayer plugin maybe? Would help in case of errors.

Reply:
Done :)